fetch-mods/get-diff.ts

```diff
@@ -51,11 +51,13 @@ export function getDiff(previousDatabase: Mod[], nextDatabase: Mod[]) {
       previousDatabaseMod?.prerelease?.version !==
         nextDatabaseMod.prerelease.version
     ) {
-      diff.push({
-        diffType: "update-prerelease",
-        previousMod: previousDatabaseMod,
-        nextMod: nextDatabaseMod,
-      });
+      if (happenedWithinDayCount(nextDatabaseMod.prerelease.date, 1)) {
+        diff.push({
+          diffType: "update-prerelease",
+          previousMod: previousDatabaseMod,
+          nextMod: nextDatabaseMod,
+        });
+      }
```

Comment:
I appreciate the fix but it is unrelated to this PR, please move it to a separate PR for better tracking. Good practice to learn anyway :)